tests/ui/c-variadic/no-closure.stderr

```diff
@@ -10,26 +10,14 @@ error: unexpected `...`
 LL |     let f = |...| {};
    |              ^^^ not a valid pattern
    |
-help: for a rest pattern, use `..` instead of `...`
-   |
-LL -     let f = |...| {};
-LL +     let f = |..| {};
-   |
+   = note: C-variadic type `...` is not allowed here
 
 error[E0743]: C-variadic type `...` may not be nested inside another type
```

Comment:
existing, but what does "may not be nested inside another type" mean here :sweat_smile: